ground_truth_labeling_jobs/multi_modal_parallel_sagemaker_labeling_workflows_with_step_functions/sagemaker_ground_truth_workflows.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>Please provide more information about the following cell. Here is a suggestion:</p><p> </p><blockquote>The following cell creates label category configuration files for the labeling jobs. These files are used to identify the label categories labelers use to annotate objects, and <em>label category attributes</em> which can be used to provide additional information about objects or scenes. For more information about these files, see [Create a Labeling Category Configuration File with Label Category and Frame Attributes](https://docs.aws.amazon.com/sagemaker/latest/dg/sms-label-cat-config-attributes.html).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>